clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java

```diff
@@ -149,7 +155,11 @@ public static void restoreProperties() {
     if (OLD_PROPERTIES != null) {
       TCProperties tcProperties = TCPropertiesImpl.getProperties();
       for (Map.Entry<String, String> entry : OLD_PROPERTIES.entrySet()) {
-        tcProperties.setProperty(entry.getKey(), entry.getValue());
+        try {
+          tcProperties.setProperty(entry.getKey(), entry.getValue());
+        } catch(NullPointerException e) {
```

Comment:
Then we're not setting the result back right?  Would seem this should be a failure?

Reply:
In fact, it will remove the value as I want. And then fail on a NPE caused by a value.trim(). I've raised Terracotta-OSS/terracotta-core#852 about it.